docs/dasharo-menu-docs/overview.md

```diff
@@ -58,6 +58,37 @@ board variant.
   used for setup application. Lists all bootable options and allows one to
   override the boot path.
 
+### Hotkeys
+
+In the standard EDK2 BIOS, the functionality of the F9 and F10 hotkeys is
```

Comment:
I do not think we need to compare with that, regular end user may not know what that means. In this documentation we simple describe what is the expected behavior in Dasharo firmware menus.

This paragraph should be rephrased, or maybe even removed if no idea how to rephrase it.

Reply:
Paragraph [removed](https://github.com/Dasharo/docs/pull/709/files#diff-256c66f72451358d31ff2d64488936acff63f53b0b36cbb8e1f0be9b9d5406d0R61)